ext/crypto/export_key.rs

```diff
@@ -168,3 +189,156 @@ fn export_key_rsa(
     }
   }
 }
+
+fn bytes_to_b64(bytes: Vec<u8>) -> String {
+  base64::encode_config(bytes, base64::URL_SAFE_NO_PAD)
+}
+
+fn export_key_ec(
+  format: ExportKeyFormat,
+  key_data: RawKeyData,
+  algorithm: ExportKeyAlgorithm,
+  named_curve: EcNamedCurve,
+) -> Result<ExportKeyResult, deno_core::anyhow::Error> {
+  match format {
+    ExportKeyFormat::Spki => {
+      let subject_public_key = match named_curve {
+        EcNamedCurve::P256 => {
+          let point = key_data.as_ec_public_key_p256()?;
+
+          point.as_ref().to_vec()
+        }
+        EcNamedCurve::P384 => {
+          let point = key_data.as_ec_public_key_p384()?;
+
+          point.as_ref().to_vec()
+        }
+        _ => return Err(data_error("Unsupported named curve")),
```

Comment:
Exhaustive matching helps catch bugs. Ditto for the `match`'s below
```suggestion
        EcNamedCurve::P521 => return Err(data_error("Unsupported named curve")),
```